### ISSUE TITLE
.se updated date parsing error

### ISSUE THREAD

**Issue by Aeonian9** [DannyCork/python-whois#302]
## Describe the bug
.se domain whois query fails to parse '-' modified date and raises UnknownDateFormat exception.

## To Reproduce
```
>>> import whois
>>> whois.query('amcon.se')
we have se
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.10/site-packages/whois/__init__.py", line 289, in query
    return Domain(
  File "/usr/local/lib/python3.10/site-packages/whois/_3_adjust.py", line 57, in __init__
    self.last_updated = str_to_date(data["updated_date"][0], self.tld.lower())
  File "/usr/local/lib/python3.10/site-packages/whois/_3_adjust.py", line 228, in str_to_date
    raise UnknownDateFormat("Unknown date format: '%s'" % text)
whois.exceptions.UnknownDateFormat: Unknown date format: '-'
```

**Comment by maarten-boot** [comment]
fixed in #303

**Comment by maarten-boot** [comment]
i expect this can be closed ?
